config/locales/views/talk/show/en.yml

```diff
@@ -0,0 +1,7 @@
+# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
+
+en:
+  talk:
+    show:
+      welcome: "Hello Everyone! Welcome to Publiclab Talk Pad"
```

Comment:
Let's go with this -- link to the wiki page and can you paste in the text from Stevie (above) onto that new wiki page? If you can rebase I'll merge this asap. Thanks!